src/System.Private.ServiceModel/src/System/ServiceModel/Security/X509CertificateInitiatorClientCredential.CoreCLR.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Additionally, this file will be going away once HttpClientHandler supports certificates.